napalm_eos/eos.py

```diff
@@ -795,9 +795,9 @@ def get_arp_table(self):
 
         return arp_table
 
-    def get_ntp_peers(self):
+    def get_ntp_stats(self):
 
-        ntp_peers = dict()
+        ntp_stats = list()
```

Comment:
I agree with you, in the grand scheme of things it's irrelevant for us. I just thought it was interesting. I used to instantiate lists and dicts in the same way as you until I read about that. So it was more a FYI than a request to change anything.